.github/workflows/release.yml

```diff
@@ -0,0 +1,162 @@
+on:
+  pull_request:
+    types:
+      - closed
+
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  TRAVIS_BRANCH: ${{ github.ref_name }}
+  GITHUB_BRANCH: ${{ github.ref_name }}
+  TRAVIS_BUILD_DIR: ${{ github.workspace }}
+  TRAVIS_COMMIT: ${{ github.sha }}
+  BUILD_ID: ${{ github.run_id }}
+  TRAVIS_RUN_NUMBER: ${{ github.run_attempt }}
+  TRAVIS_BUILD_NUMBER: ${{ github.run_id }}
+  JOB_ID: ${GITHUB_RUN_ID}
+  PROXY_HOST_BPM: ${{ secrets.E2E_HOST }}
+  E2E_HOST_APA: ${{ secrets.E2E_HOST_APA }}
+  E2E_HOST: ${{ secrets.E2E_HOST }}
+  E2E_USERNAME: ${{ secrets.E2E_ADMIN_EMAIL_IDENTITY }}
+  E2E_PASSWORD: ${{ secrets.E2E_PASSWORD }}
+  E2E_ADMIN_EMAIL_IDENTITY: ${{ secrets.E2E_ADMIN_EMAIL_IDENTITY }}
+  E2E_ADMIN_PASSWORD_IDENTITY: ${{ secrets.E2E_ADMIN_PASSWORD_IDENTITY }}
+  USERNAME_ADF: ${{ secrets.E2E_USERNAME }}
+  PASSWORD_ADF: ${{ secrets.E2E_PASSWORD }}
+  URL_HOST_ADF: ${{ secrets.URL_HOST_ADF }}
+  IDENTITY_ADMIN_EMAIL: ${{ secrets.E2E_ADMIN_EMAIL_IDENTITY }}
+  IDENTITY_ADMIN_PASSWORD: ${{ secrets.E2E_ADMIN_PASSWORD_IDENTITY }}
+  AWS_S3_BUCKET_ACTIVITI_LICENSE: ${{ secrets.AWS_S3_BUCKET_ACTIVITI_LICENSE }}
+  HOST_SSO: ${{ secrets.HOST_SSO }}
+  LOG_LEVEL: "ERROR"
+  E2E_LOG_LEVEL: "ERROR"
+  E2E_MODELER_USERNAME: ${{ secrets.E2E_MODELER_USERNAME }}
```

Comment:
probably not neeeed

Reply:
some tests were failing because they were trying to login as  modeler